Server/Server.Core/Server.Core.Static/Server.cpp

```diff
@@ -1,635 +0,0 @@
-#include <future>
```

Comment:
What's the need for removing .cpp file?

Reply:
The Server class template appeared, the implementation moved to .hpp.

Reply:
We discussed it, there is no need to use the template parameter. It also breaks compatibility with the branch where the server builder is being developed. Please keep your development in line with others.

Reply:
We discussed it, there is no need to use the template parameter. It also breaks compatibility with the branch where the server builder is being developed. Please keep your development in line with others.

Reply:
> We discussed it, there is no need to use the template parameter. It also breaks compatibility with the branch where the server builder is being developed. Please keep your development in line with others.

The point is that my changes are already ready.
I need a template parameter for a method/function. I see no reason not to run my code, if it works correctly. What you write and it will affect you when you merge it, that's ok, that's fine.

Reply:
> > We discussed it, there is no need to use the template parameter. It also breaks compatibility with the branch where the server builder is being developed. Please keep your development in line with others.
> 
> The point is that my changes are already ready. I need a template parameter for a method/function. I see no reason not to run my code, if it works correctly. What you write and it will affect you when you merge it, that's ok.

Both yours and @Orange-hanter 's changes are not yet made - since they're not in the dev branch.

Reply:
Also, it's not a matter of who did it first, it's a matter of which one will be used in the future.
Since there clearly is a disagreement, I suggest we discuss this on today's meeting or some time in the closest future.
@memorisecodead @Orange-hanter please inform if you can join today, on Wednesday or some other time this week.

Reply:
well, it appears I won't =(
How about Wednesday though?

Reply:
> well, it appears I won't =( How about Wednesday though?

No problem, then we'll discuss it on Wednesday :)